tree/ntuple/v7/src/RPageStorage.cxx

```diff
@@ -390,7 +390,7 @@ void ROOT::Experimental::Internal::RPageSource::EnableDefaultMetrics(const std::
       *fMetrics.MakeCounter<Detail::RNTupleAtomicCounter *>("szUnzip", "B", "volume after unzipping"),
       *fMetrics.MakeCounter<Detail::RNTupleAtomicCounter *>("nClusterLoaded", "",
                                                             "number of partial clusters preloaded from storage"),
-      *fMetrics.MakeCounter<Detail::RNTupleAtomicCounter *>("nPageLoaded", "", "number of pages loaded from storage"),
+      *fMetrics.MakeCounter<Detail::RNTupleAtomicCounter *>("nPageRead", "", "number of pages read from storage"),
       *fMetrics.MakeCounter<Detail::RNTupleAtomicCounter *>("nPagePopulated", "", "number of populated pages"),
```

Comment:
I think this should rename `nPagePopulated` to `nPageRead` and leave `nPageLoaded`. Which poses the question what is the difference between "reading" and "loading", and whether we increment the right counters in `LoadPage`...

Reply:
Right, that's not obvious. It should be like this: `reading` means reading the (sealed) page from disk. And `loading` is reading + unsealing.